v2/cards/bmi_buffer.json

```diff
@@ -10,5 +10,12 @@
   "stripped_text": "Whenever a program is trashed from your grip, host it on BMI Buffer instead of adding it to your heap. click click: Install 1 hosted program (paying all costs).",
   "stripped_title": "BMI Buffer",
   "text": "Whenever a program is trashed from your grip, host it on BMI Buffer instead of adding it to your heap.\n[click][click]: Install 1 hosted program (paying all costs).",
-  "title": "BMI Buffer"
+  "title": "BMI Buffer",
+  "layout_id": "progression",
+  "sides": [
```

Comment:
i'm torn on having the default for cards like this having some parts outside of the sides and some parts inside.

Reply:
Part of why I did it this way was to reduce the need to acknowledge extra sides when reading from the json. If, for some reason, we don't then there's still a complete card defined outside of the sides.

It also reduces the duplicate data (since the base card defines everything and the sides overwrite only what changes. It also means the overall structure of the `sides` attribute doesn't change between layout types.

I'm not actually wholly against separating the base version into its own side, but I can't think of any reason why that would allow us to define something we can't already, and once it's in the API we can format it however we want.

Reply:
yeah.  I think "net less work overall" is the right way to resolve this.  :)

Handling the schema files to support "empty fields unless they are specified in every side" is ... gonna be super annoying.

Also, you reminded me that this does minimize work on existing readers, so huzzah!